### ISSUE TITLE
Questions regarding poppler-qt5

### ISSUE THREAD

**Issue by gojpdchx** [BasioMeusPuga/Lector#74]
1. In README.md, poppler-qt5 and python-poppler-qt5 are listed as optional. Which function(s) depend(s) on these two? Are they used for opening / rendering PDF files?
2. Could you please change the PKGBUILDs of the AUR packages lector and lector-git such that poppler-qt5 and python-poppler-qt5 become **optional** dependencies? The current PKGBUILDs define these two as compulsory dependencies.

**Comment by BasioMeusPuga** [comment]
Yes. poppler is the rendering library, python-poppler-qt5 provides the python bindings.
They're optional in the sense that the program will continue to work without them. I've purposely included them in the `depends` list in the PKGBUILD because it reduces hassle since any AUR helper will also install python-poppler-qt5.
